rails-snow-queen/app/assets/javascripts/quote.js

```diff
@@ -35,23 +36,38 @@ class Quote {
       secondaryAddressNode.appendChild(div);
     }); 
 
-    const areaNode = document.getElementById("areaModal");
-    areaNode.innerText = `${quoteData.totalAreaInSqFt.toLocaleString(undefined, {maximumFractionDigits: 0})}`;
-    
-    const totalDueNode = document.getElementById("totalModal");
-    totalDueNode.innerText = `${quoteData.totalDue.toLocaleString(undefined, {maximumFractionDigits: 2})}`;
-    
-    const addressNodeForm = document.getElementById("quote_address");
-    addressNodeForm.value = quoteData.geocodedAddress;
-    
-    const areaNodeForm = document.getElementById("quote_area");
-    areaNodeForm.value = quoteData.totalAreaInSqFt;
+    this.updateStaticMap(quoteData.polygons);
 
-    const totalDueNodeForm = document.getElementById("quote_total");
-    totalDueNodeForm.value = quoteData.totalDue;
+    document.getElementById("areaModal").innerText = `${quoteData.totalAreaInSqFt.toLocaleString(undefined, {maximumFractionDigits: 0})}`;
+    document.getElementById("totalModal").innerText = `${quoteData.totalDue.toLocaleString(undefined, {maximumFractionDigits: 2})}`;
+    document.getElementById("quote_address").value = quoteData.geocodedAddress;
+    document.getElementById("quote_area").value = quoteData.totalAreaInSqFt;
+    document.getElementById("quote_total").value = quoteData.totalDue;
+    document.getElementById("quote_polygons").value = polygonsLatLngs;
+    document.getElementById("quote_static_map_URL").value = this.staticMapURL;
```

Comment:
Hahaha nice hack! You don't actually need to store this in the DB, you can build it in Ruby from the polygons, but will do for today ;)